lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -433,7 +433,7 @@
                   <p>Copy and paste your share link:</p>
                   <code><?php print $custom_organ_share_link ?></code>
                   <div class="form-item -padded submit-done-container">
-                    <input type="submit" class="button submit-done" value="Done" />
+                    <input type="submit" class="button js-close-modal" value="Done" />
```

Comment:
Again, we might want to add some clarity about whether we want to log skipping the form, and getting through the flow and clicking done, differently.